package.json

```diff
@@ -21,71 +21,79 @@
       "underscore": "lodash"
     }
   },
+  "ava": {
```

Comment:
Wait for it then. Would you like me to feed this back to goblindegook/genesis-starter?